crates/subspace-node/src/chain_spec.rs

```diff
@@ -78,7 +144,16 @@ pub fn testnet_config() -> Result<ChainSpec, String> {
         // Protocol ID
         Some("subspace"),
```

Comment:
Not sure we want this.

```suggestion
        Some("aries"),
```

Reply:
This will make polkadot.js frontend incompatible. I'm also not sure if we should have it different for every iteration of the testnet.

Reply:
Usually, we won't have many :D Anyway, not a big deal, it's fine to just keep it as is.